Jenkinsfile

```diff
@@ -6,12 +6,27 @@ pipeline {
         stage ('Docker') {
             agent {
                 dockerfile {
-                    args '-e HOME=/tmp -e BUILD_CONTEXT=ci'
+                    args '-e HOME=/tmp -e BUILD_CONTEXT=ci --cap-add=NET_ADMIN -u 0:0'
```

Comment:
could you adjust caps until you can add/configure the vcan adapter without getting root?

Reply:
with network host mode you can use can adapter from the host.

https://forums.docker.com/t/creating-and-sharing-virtualcan-network-interface-between-docker-containers/61596

Reply:
for caps, start with `--net=host and --cap-add=ALL`